notebooks/05_operationalize/lightgbm_criteo_o16n.ipynb

```diff
@@ -0,0 +1,785 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Copyright (c) Microsoft Corporation. All rights reserved.\n",
+    "\n",
+    "Licensed under the MIT License."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Goal\n",
```

Comment:
Maybe a better title is needed here.

Reply:
ok, updating this section a bit

Reply:
we can add the svg that was going to the document. I think @jreynolds01 is going to work on this PR this week, I can update it tomorrow

Reply:
The diagram is already updated